### DISCUSSION TITLE
[SUGGESTION] - Manage Github projects from the mobile Client

### DISCUSSION THREAD

**Discussion by BrentMifsud** [community/community#4617] [Mobile] [answered]
Currently there is very little we can do regarding projects in the mobile app.

Some features I would like to see:
- assign issues to projects
- change an issues column (todo, in progress, done, etc...)
- create/update/delete projects

Nice to haves:
- Kanban board similar to the website. (maybe an iPad only feature?)

**Comment by smooth80** [comment]
client connect natively

**Comment by mxie** [comment, marked as answer]
Thanks for your feedback, @BrentMifsud! Check out [this discussion](https://github.com/github/feedback/discussions/3023) about this feature request and provide any additional feedback that isn't already covered. This will help us understand your use case.
